sway-core/src/semantic_analysis/namespace/namespace.rs

```diff
@@ -286,6 +290,46 @@ impl Namespace {
             parent_mod_path,
         }
     }
+
+    #[allow(clippy::too_many_arguments)]
+    pub(crate) fn insert_trait_implementation(
```

Comment:
`insert_trait_implementation` and  `get_methods_for_type_and_trait_name` now use `trait_name.to_fullpath(self)` which requires the self to be a `Namespace` instead of `Items` so they were moved to `Namespace`.

Reply:
SGTM